src/shrpx.cc

```diff
@@ -386,6 +403,9 @@ void exec_binary() {
     if (pid == -1) {
       auto error = errno;
       LOG(ERROR) << "fork() failed errno=" << error;
+    } else {
+      // update PID tracking information in systemd
+      shrpx_sd_notifyf(0, "MAINPID=%s\n", util::utos(pid).c_str());
```

Comment:
Is this required since we eventually do that in the new master process (line1279)?